src/components/Content/Pursuance/PursuanceMenu.js

```diff
@@ -18,6 +19,13 @@ const PursuanceMenu = ({ currentPursuanceId, location }) => {
       <div>
         <PursuanceMenuItem
           className="pursuance-top-btn"
+          pursuanceId={currentPursuanceId}
+          location={location}
+          label='About'
+          action='about'
+          icon={<Info size={28} />}
+        />
+        <PursuanceMenuItem
```

Comment:
@4xdk The latest version of this (in `develop`) is different; we no longer need to pass in `pursuanceId` and `location` explicitly, as `PursuanceMenuItem` is now reduxified.